apps/jira/jira-app/src/components/App/Steps/InstanceStep.tsx

```diff
@@ -61,51 +38,52 @@ export default class InstanceStep extends React.Component<Props, State> {
         <Heading>Configure</Heading>
         <Paragraph>Select the Jira site and project you want to connect</Paragraph>
         <div className="jira-config" data-test-id="instance-step">
-          <Select
-            testId="instance-selector"
-            className="selector"
-            // @ts-ignore: 2339
-            onChange={(e) => pickResource(e.target.value)}
-            isDisabled={resources.length === 1}
-            value={selectedResource || ''}>
-            <Option value="">Select a site</Option>
-            {resources.map((r) => (
-              <Option key={r.id} value={r.id}>
-                {r.url.replace('https://', '')}
-              </Option>
-            ))}
-          </Select>
+          <div style={{ display: 'flex', gap: 16, alignItems: 'center', width: '100%' }}>
+            <Select
+              testId="instance-selector"
+              className="selector"
+              style={{ minWidth: 320, maxWidth: 400, flex: 1 }}
+              onChange={(e: React.ChangeEvent<HTMLSelectElement>) => pickResource(e.target.value)}
+              isDisabled={resources.length === 1}
+              value={selectedResource || ''}>
+              <Option value="">Select a site</Option>
+              {resources.map((r) => (
+                <Option key={r.id} value={r.id}>
+                  {r.url.replace('https://', '')}
+                </Option>
+              ))}
+            </Select>
 
-          <div className="search-projects">
-            <TextInput
-              width="full"
-              placeholder={selectedProject ? selectedProject.name : 'Search for a project'}
-              value={this.state.inputValue}
-              onChange={(ev) => {
-                ev.persist();
-                this.handleInputChange(ev);
-              }}
-              onFocus={() => {
-                this.setState({ inputValue: '' });
+            <Autocomplete<CloudProject>
```

Comment:
Nice update to add Autocomplete. I thought this `<CloudProject>` looked out of place at first, but learned from the docs that's how we can set the type for items 🔥